### DISCUSSION TITLE
Quwashop

### DISCUSSION THREAD

**Discussion by andikctistian** [community/community#59378] [Programming Help]
#!/bin/bash

# Runs tests for all supported PHP versions
# Usage examples:
# - ./run-tests-for-all-php-versions.sh --group 20221124
# - ./run-tests-for-all-php-versions.sh --exclude-group slow

docker-compose run php71 ./run-tests.sh $@ && \
docker-compose run php72 ./run-tests.sh $@ && \
docker-compose run php73 ./run-tests.sh $@ && \
docker-compose run php74 ./run-tests.sh $@ && \
docker-compose run php80 ./run-tests.sh $@ && \
docker-compose run php81 ./run-tests.sh $@

**Comment by andikctistian** [comment]
Good

**Comment by queenofcorgis** [comment]
Hi @andikctistian , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.
